### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by septs**
original link 404:
https://github.com/tc39/ecmascript-asset-references

### CODE REVIEW

**Comment by septs** [comment]
CC @ljharb request merge the pr

**Comment by septs** [comment]
se https://github.com/JSCIG/dataset/runs/1168936946?check_suite_focus=true

File: README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Note that as part of the onboarding process your repository name may be normaliz
 [class-fields-notes]: https://github.com/tc39/notes/blob/master/meetings/2020-03/march-31.md#add-support-for-optionalchainprivateidentifier-in-class-features-proposals
 [private-fields-and-methods-refresher-notes]: https://github.com/tc39/notes/blob/master/meetings/2019-01/jan-30.md#private-fields-and-methods-refresher
 [tests-class-fields]: https://github.com/tc39/test262/pulls?page=1&q=is%3Apr+is%3Aclosed+private+fields
-[tests-static-class-fields]: https://github.com/tc39/test262/pulls?q=is%3Apr+is%3Aclosed+static+fields+
```
Comment (ljharb) [inline README.md:85]: these links provide identical content

**Review by ljharb — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (septs) [reply README.md:85]: last `+` is unnecessary spaces

**Comment by septs** [comment]
@ljharb thanks

https://github.com/JSCIG/dataset/runs/1169184316?check_suite_focus=true

by `JSCIG/dataset` project, errors found all clear.

File: README.md (hunk 1 above)
Comment (ljharb) [reply README.md:85]: Sure, but it's a URL, as long as it leads to the correct content it doesn't matter.

**Pull request merged**